Assets/StreamingAssets/Factions/FACTION.TXT

```diff
@@ -649,7 +649,7 @@ maxf: 90
 		summon: -1
 		region: -1
 		power: 25
-		flags: 0
+		flags: 256
```

Comment:
Not sure it was necessary to cover the religious knights, but I guess it doesn't hurt